docker-compose.example.yml

```diff
@@ -13,14 +14,20 @@ services:
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock"
       - "/var/lib/docker/containers/:/var/lib/docker/containers/"
-      - "/var/lib/pterodactyl/:/var/lib/pterodactyl/"
+      - "/etc/pterodactyl/:/etc/pterodactyl/"
+      - "/var/log/pterodactyl/:/var/log/pterodactyl/"
       - "/srv/daemon-data/:/srv/daemon-data/"
```

Comment:
`/srv/daemon-data` is deprecated and only used by old nodes now. `/var/lib/pterodactyl/voulumes` replaces it.